app/assets/javascripts/components/story/StoryDescriptionIcon.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const StoryDescriptionIcon = ({ description }) => {
+  if ( description ) {
+    return (
+      <span className='Story__description-icon' >
+        <i className={`mi md-dark md-16 question_answer`}>question_answer</i>
```

Comment:
You don't need theses curly braces around the class name 😄

Reply:
I'm slowly losing this addiction :D